test/controllers/name_controller_test.rb

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Interesting.  It's actually my editor, emacs, that adds it.  I just tried deleting it and it re-added it.  I'm sure I can figure out how to delete it if anyone cares.  Otherwise I'm just going to leave it the way it is.